terrain_planner_benchmark/src/test_ics.cpp

```diff
@@ -164,8 +164,8 @@ int main(int argc, char** argv) {
 
   auto grid_map_pub_ = nh.advertise<grid_map_msgs::GridMap>("grid_map", 1, true);
   auto reference_map_pub_ = nh.advertise<grid_map_msgs::GridMap>("reference_map", 1, true);
-  auto yaw_pub = nh.advertise<visualization_msgs::Marker>("yaw", 1, true);
```

Comment:
Some of these changes are find+replace, so more are needed to get these other tests and tools to compile. I'll get them ported after this PR goes in.